waspc/src/Wasp/Generator/ServerGenerator/Common.hs

```diff
@@ -85,3 +87,6 @@ serverTemplatesDirInTemplatesDir = [reldir|server|]
 
 srcDirInServerTemplatesDir :: Path' (Rel ServerTemplatesDir) (Dir ServerTemplatesSrcDir)
 srcDirInServerTemplatesDir = [reldir|src|]
+
+entityNameToPrismaIdentifier :: String -> String
+entityNameToPrismaIdentifier entityName = toLower (head entityName) : tail entityName
```

Comment:
Decided to extract this since it was used in both `ServerGenerator/JobGenerator.hs` and `ServerGenerator/OperationsG.hs`. It also seemed pretty important if it should need to change in the future, we'd want just one place to do so I think.

Reply:
Makes sense, will add a comment providing a bit of info/context! 👍🏻